schema/pil.js

```diff
@@ -35,7 +35,8 @@ class PIL extends BaseModel {
           items: { type: 'string' }
         },
         notesCatD: { type: ['string', 'null'] },
-        notesCatF: { type: ['string', 'null'] }
+        notesCatF: { type: ['string', 'null'] },
+        billable: { type: 'boolean' }
```

Comment:
I don't think this can be a property of the PIL because the times when it's false will mostly likely be where it is billed for two establishments and should only be billable for one. Also the same PIL may be billable and not billable in different years.